### ISSUE TITLE
Auto-nest doesn't work in VS2015

### ISSUE THREAD

**Issue by MarioHsiao** [madskristensen/FileNesting#63]
### Installed product versions
- Microsoft Visual Studio Enterprise 2015, Version 14.0.25431.01 Update 3
- File Nesting, Version 2.5.63, Last updated 9/9/2016

### Description
Enable automatic nesting doesn't work.
Auto-nest selected items doesn't work.

### Steps to recreate
1. Create a new console application project.
2. Add another .config file named App.Debug.config
3. Right click project select File Nesting - Enable automatic nesting, doesn't work.
4. Select Auto-nest selected items doesn't work.

**Comment by michael-hawker** [comment]
Yeah, I saw this as well with File Nesting v2.6.67 on VS2015 Update 3.  I selected 9 files (.ts with associated .js and .js.map files) in my Universal Windows web app project.  I selected auto-nest selected items, and VS just hung for a minute before doing nothing...
